examples/next/pages/examples/listings/ExperienceCard.tsx

```diff
@@ -4,7 +4,7 @@ export const ExperienceCard = ({ img, title }) => (
   <Card
     width="20rem"
     padding="0"
-    borderRadius={theme.radii.large}
+    borderRadius={theme.radii.large.value}
```

Comment:
Note: temporarily adding `.value` to get the example to work. In the next PR I will address this and the reference issue so users won't have to add `.value`.